ssl/s3_lib.c

```diff
@@ -4819,9 +4837,16 @@ EVP_PKEY *ssl_generate_param_group(SSL *s, uint16_t id)
      * s->ctx->libctx and s->ctx->propq when paramgen has been updated to be
      * provider aware.
      */
+#if 0
+    pkey_ctx_name = (ginf->flags & TLS_GROUP_FFDHE) ? "DH" : "EC";
```

Comment:
I forget if we want a `!= 0` check for the flags, in our style guide.

Reply:
Yes, we should. Done.